presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -472,6 +472,20 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
     }
 
     @Override
+    public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, String column)
+    {
+        RaptorTableHandle table = (RaptorTableHandle) tableHandle;
+        List<TableColumn> existingColumns = dao.listTableColumns(table.getSchemaName(), table.getTableName());
+        if (existingColumns.size() == 1) {
```

Comment:
For Raptor, we don't keep track of the `column_id` (we take the max existing value when adding a new column), so we can't allow dropping the right-most (highest ID) column of the table.